lib/Target/ExportSystemC/Patterns/SystemCEmissionPatterns.cpp

```diff
@@ -18,6 +18,17 @@ using namespace circt;
 using namespace circt::systemc;
 using namespace circt::ExportSystemC;
 
+static void parenthesizeOnLowerPrecedence(InlineEmitter &emitter,
+                                          Precedence prec, EmissionPrinter &p) {
+  if (emitter.getPrecedence() >= prec)
```

Comment:
It may be cleaner to store `emitter.getPrecedence() >= prec` in a temporary due to its use in two places.